docs/s3.md

```diff
@@ -34,55 +34,63 @@ leads to an HTML or .txt file, it will get rendered.  And if it leads to any oth
 
 ## Developer access/use
 
-The bucket is owned by the Developer user for our ESO account which we should all have access
-to.  Use that from the aws console to manipulate anything about how the bucket is setup. 
+The bucket is owned by the Developer role for our ESO account which we should all have access
+to.  Use that from the aws console to manipulate anything about how the bucket is setup.
 
 ### Install the aws cli
 
-Follow the 
-[Installing AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) 
+Follow the
+[Installing AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
 instructions for the machine you want to access s3 from.
 
 ### Credentials
 
-IT set up a service account for us to use when accessing the bucket via the cli.  They
-added the credentials to the 1password SLV team vault as the S3 service account login.
- 
+Use the puppet-bastion account with the eso profile to connect provide aws cli
```

Comment:
This sentence is slightly confusing ("to connect provide..."); it may just need an extra comma or two or slight rewording to make it more clear.

Reply:
:man_facepalming: Me learn English one day.  Thanks @billclaytor .  I have pushed an update that should be somewhat more comprehensible.